sklearn/manifold/tests/test_spectral_embedding.py

```diff
@@ -1,6 +1,6 @@
+import pytest
```

Comment:
wait that's the first explicit dependency on pytest, right?

Reply:
nop, it has been discuss in another PR:
https://github.com/scikit-learn/scikit-learn/pull/10081#issuecomment-349030470

We also added there: https://github.com/scikit-learn/scikit-learn/blob/4f710cdd088aa8851e8b049e4faafa03767fda10/sklearn/preprocessing/tests/test_target.py

There is some in the ColumnTransformer.

Are you welling to revert it?

Reply:
No that's good, sorry I'm out of the loop.

Reply:
we should have ping you there thought

Reply:
I'm sure that wouldn't have gotten lost in my 17615 unread scikit-learn notifications ;) Hm can I filter stuff that I'm pinged in... good question...

Reply:
oh that reduces it to only 5782 (only unread threads that I'm subscribed to)!